bin/quiesce-mariadb.sh

```diff
@@ -66,8 +66,6 @@ function getMysqlConnectionArgs()
     local arg
     arg+=" "$(getConfValue "$config" "$dbname-admin-user" "--user=")
     arg+=" "$(getConfValue "$config" "$dbname-admin-password" "--password=")
-    arg+=" "$(getConfValue "$config" "$dbname-host" "--host=")
-    arg+=" "$(getConfValue "$config" "$dbname-port" "--port=")
```

Comment:
The root user doesn't have a password, so I think the admin-password option should be excluded.

Reply:
If the script is smart enough to exclude the `--password` option if the admin password is blank, then it's fine.

Reply:
it works even if you put empty password, I checked that before pushing changes